cmd/get/get_test.go

```diff
@@ -0,0 +1,69 @@
+package get_test
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+
+	"github.com/puppetlabs/prm/cmd/get"
+	"github.com/stretchr/testify/assert"
+)
+
+type test struct {
+	name           string
+	args           []string
+	expectedErrMsg string
+	expectedStdOut string
+}
+
+func Test_GetCommand(t *testing.T) {
+	tests := []test{
+		{
+			name:           "Should display help when no subcommand passed to 'get'",
+			args:           []string{""},
+			expectedErrMsg: "Displays the requested configuration value",
```

Comment:
Should this be `expectedMsg` instead of `expectedErrMsg`?

Reply:
or is this bc the info is returned via stderr? that seems weird

Reply:
I can rename those, actually, similar to how I did with the `prm get` tests